TrafficCapture/trafficReplayer/src/test/java/org/opensearch/migrations/replay/TrafficStreamGenerator.java

```diff
@@ -0,0 +1,258 @@
+package org.opensearch.migrations.replay;
+
+import lombok.AllArgsConstructor;
+import lombok.SneakyThrows;
+import lombok.extern.slf4j.Slf4j;
+import org.opensearch.migrations.trafficcapture.protos.TrafficObservation;
+import org.opensearch.migrations.trafficcapture.protos.TrafficStream;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Random;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.function.Supplier;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+@Slf4j
+public class TrafficStreamGenerator {
```

Comment:
It would be good to know how this operates at a high level. It seems like it could be very helpful, does this create traffic streams with any sequence of TrafficObservations or is there sanity checks to make more legitimate streams?

Reply:
Totally agree.  I'll put at least a TODO on it.  The truth is that this thing is just a very specific noise machine at the moment, but it's closer to being a great utility than I had originally realized.